feature-libs/quote/components/quote-confirm-action-dialog/quote-confirm-action-dialog.component.html

```diff
@@ -0,0 +1,107 @@
+<div
+  #dialog
+  class="cx-modal-container"
+  [cxFocus]="focusConfig"
+  (esc)="dismissModal('Escape clicked')"
+>
+  <div
+    class="cx-modal-content"
+    *ngIf="confirmationContext$ | async as confirmationContext"
+  >
+    <!-- Modal Header -->
+
+    <div class="cx-dialog-header modal-header">
+      <div class="cx-dialog-title modal-title">
+        {{
+          confirmationContext.title
+            | cxTranslate: { code: confirmationContext.quote.code }
+        }}
+      </div>
+      <button
+        type="button"
+        class="close"
+        [attr.aria-label]="'common.close' | cxTranslate"
+        (click)="dismissModal('Cross click')"
+      >
+        <span aria-hidden="true">
+          <cx-icon [type]="iconTypes.CLOSE"></cx-icon>
+        </span>
+      </button>
+    </div>
+
+    <!-- Modal Body -->
+    <div class="cx-dialog-body modal-body">
+      <div class="cx-dialog-row">
+        <div class="cx-dialog-item">
+          <div class="quote-name-description">
+            <div class="quote-name">
+              <div class="heading">
+                {{ 'quote.confirmActionDialog.name.heading' | cxTranslate }}
+              </div>
+              <div class="content">
+                {{
+                  'quote.confirmActionDialog.name.content'
+                    | cxTranslate: { name: confirmationContext.quote.name }
+                }}
+              </div>
+            </div>
+            <div
+              class="quote-description"
+              *ngIf="confirmationContext.quote.description"
+            >
+              <div class="heading">
+                {{
+                  'quote.confirmActionDialog.description.heading' | cxTranslate
+                }}
+              </div>
+              <div class="content">
+                {{
+                  'quote.confirmActionDialog.description.content'
+                    | cxTranslate
+                      : { description: confirmationContext.quote.description }
+                }}
```

Comment:
```suggestion
                {{ description: confirmationContext.quote.description }}
```
please remove pipe and i18n key - there is no need to translate it

Reply:
done:
{{ confirmationContext.quote.description }}